python/lsst/pipe/tasks/ingest.py

```diff
@@ -42,7 +42,7 @@ class ParseConfig(Config):
                             doc="Properties and name of translator method")
     defaults = DictField(keytype=str, itemtype=str, default={},
                          doc="Default values if header is not present")
-    hdu = Field(dtype=int, default=0, doc="HDU to read for metadata")
+    hdu = Field(dtype=int, default=-(1 << 31), doc="HDU to read for metadata") # default == INT_MIN
```

Comment:
Is this assuming something about `sizeof(int)`?  Can you get the default value out of C++-land?